scripts/deploy.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e -u -x
+set -o pipefail
+source /opt/rpc-extras/os-ansible-deployment/scripts/scripts-library.sh
+
+export RPCD_LOGGING=${RPCD_LOGGING:-"FALSE"}
+export RPCD_AIO=${RPCD_AIO:-"FALSE"}
+
+OSAD_DIR='/opt/rpc-extras/os-ansible-deployment'
+RPCD_DIR='/opt/rpc-extras/rpcd'
+
+# setup the things
+cd "${OSAD_DIR}"
+if [[ "${RPCD_AIO}" == "yes" ]]; then
+  if [[ ! -d /etc/openstack_deploy/ ]]; then
+    ./scripts/bootstrap-aio.sh
+    cp -R "${RPCD_DIR}"/etc/openstack_deploy/* /etc/openstack_deploy/
+  fi
+fi
+
+# bootstrap ansible only if not installed
+which openstack-ansible
+if [[ $? -ne 0 ]]; then
+  ./scripts/bootstrap-ansible.sh
+fi
+
+# only set up the passwords once
```

Comment:
What about putting the RPCD_LOGGING conditional stuff around this? Partly because it doesn't need to run if that's not equal to "yes" and partly to make the relationship.